### DISCUSSION TITLE
prefab vs cmake

### DISCUSSION THREAD

**Discussion by paulorrfilho** [google/oboe#1210] [Q&A] [answered]
Hi, I am in the process of porting my second music app to Android. The first one I build Oboe from the source however, I noticed that Oboe is now available via prefab. Is it stable? I read somewhere it was buggy.

thank you

**Comment by dturner** [comment, marked as answer]
Who said it was buggy??! 😉 

It's stable, easy and recommended if you don't want to make modifications to the Oboe source. 

https://github.com/google/oboe/blob/master/docs/GettingStarted.md#option-1-using-pre-built-binaries-and-headers
